ci/keystone/install.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 set -ex
 
-pip install .[keystone-auth-backend,keystone-cli]
```

Comment:
This referring to the extra in our `setup.py`, so yeah, it's supposed to be `keystone-client`. You could concievably want one and not the other, since the latter is used by our client library & command line tool, while the former is used by the server.

I think we should put this back, fixing the name.

Reply:
I goofed up in the commit message for that one. What I meant was, since we are installing keystonemiddleware for our ci tests, it will also install python-keystoneclient. So installing keystone-client is redundant for our ci tests. But it's still left in our setup.py because, like you pointed out, we might need it for client library or cli.

Reply:
Hmm. I guess I don't like leaving it out, since the original expresses intent more clearly; it's kindof a coincidence that it works without it.